corehq/apps/registration/forms.py

```diff
@@ -575,8 +584,7 @@ def __init__(self, data=None, is_add_user=None,
             ),
             hqcrispy.FormActions(
                 twbscrispy.StrictButton(
-                    (gettext("Add User") if is_add_user
-                     else gettext("Send Invite")),
+                    (gettext("Add User") if is_add_user else gettext(save_button_text)),
```

Comment:
passing a variable to `gettext` will only translate that var if it's already marked for translation elsewhere, such as with `gettext_noop`.  Probably most straightforward to just move the `gettext` call to where those strings are defined in this case.